.github/fabricbot.json

```diff
@@ -1508,9 +1513,28 @@
           ]
         },
         {
-          "label": "blazor-wasm",
+          "label": "area-grpc",
           "pathFilter": [
-            "src/ProjectTemplates/BlazorWasm.ProjectTemplates/"
+            "src/Grpc/"
+          ]
+        },
+        {
+          "label": "area-web-frameworks",
+          "pathFilter": [
+            "src/Mvc/Mvc.Abstractions/",
+            "src/Mvc/Mvc.Analyzers/",
+            "src/Mvc/Mvc.Api.Analyzers/",
+            "src/Mvc/Mvc.ApiExplorer/",
+            "src/Mvc/Mvc.Core/",
+            "src/Mvc/Mvc.Cors/",
+            "src/Mvc/Mvc.Formatters.Json/",
+            "src/Mvc/Mvc.Formatters.Xml/",
+            "src/Mvc/Mvc.NewtonsoftJson/",
+            "src/Mvc/Mvc.Testing.Tasks/src/",
+            "src/Mvc/Mvc.Testing/",
+            "src/Mvc/Mvc/",
+            "src/Mvc/perf/",
+            "src/Mvc/"
```

Comment:
Removing the root folder as I am afraid it may result either in earlier rules not working or duplicated labels. Most of the folders have been mapped already anyway. We can expand individual mappings as we find them missing.